common/locales/en/gear.json

```diff
@@ -1,7 +1,10 @@
 {
   "set": "Set",
+  "equipmentType" : "Equipment Type",
+  "klazz": "Class",
+  "groupingBy": "Grouping By ",
 
-  "weapon": "weapon",
+  "weapon": "Weapon",
```

Comment:
Won't this change every other instance that uses the `weapon` string? Have you audited that every other instance that uses it is appropriate to use the capitalized version?